packages/react/.storybook/webpack.config.js

```diff
@@ -48,13 +52,31 @@ const styleLoaders = [
   {
     loader: 'sass-loader',
     options: {
-      includePaths: [path.resolve(__dirname, '..', 'node_modules')],
-      data: `
-        $feature-flags: (
-          ui-shell: true,
-          enable-css-custom-properties: true,
-        );
-      `,
+      prependData(loaderContext) {
+        const entrypoint = path.resolve(__dirname, './_container.scss');
+        const flags = `
+          $feature-flags: (
+            ui-shell: true,
+            enable-css-custom-properties: ${CARBON_REACT_STORYBOOK_USE_CUSTOM_PROPERTIES},
+          );
+        `;
+
+        if (loaderContext.resourcePath === entrypoint) {
```

Comment:
Could you add some comments around the conditional loading of flags? not sure what this is for or when we'd enable one or the other.